### PR TITLE
Mint signup  page

### PR DESCRIPTION
**Pull request by kiptubei**
In this project, we duplicated the [mint.com ](https://accounts.intuit.com/signup.html?offering_id=Intuit.ifs.mint&namespace_id=50000026&redirect_url=https%3A%2F%2Fmint.intuit.com%2Foverview.event%3Ftask%3DS) sign up page as indicated in the readme file.

Actions:
Created two css files

- reset.css -CSSE resets to create blank/uniform starting template for purposes of multi-browser compatibility
- style.css-Main styling sheet for the project

Created one html file

- index.html - this is a single page html project and all content  and viewing tags are contained here.

### CODE REVIEW

**Comment by sumancrest0001** [comment]
### This project is under review. Please keep patience.  :pray:  :pray:
**TSE: Suman Shrestha**

File: index.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+                <p>Standard call messaging or data rates may apply</p>
+              </fieldset>
+              <fieldset>
+                <legend>Password</legend>
+                <input type="password" name="password" id="password" />
+              </fieldset>
+              <fieldset>
+                <button class="input-btn-div">
+                  <i class="fa fa-lock"></i> Create Account
+                </button>
```
Comment (sumancrest0001) [inline index.html]: [REQUIRED] Type should be 'submit' for the button. [reference](https://www.w3schools.com/tags/att_input_type_submit.asp)

**Comment by sumancrest0001** [comment]
[REQUIRED] You forgot to add a stickler file on the project. [requirement](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/)
Please add stickler and linter files and remove all the errors before submitting the project for review.

**Comment by kiptubei** [comment]
Hello Sman,

Thank you, for the review , the reason our repository did not include a stylelint.config.js file earlier is because I use Atom and my coding partner uses VSCode , the way linter is configured on the 2 IDE's differs slightly , thus explicitly including this file in the project directory causes conflicts in Atom. As such we had earlier specifically excluded stylelint.config.js from the project structure in gitignore. though locally we use the tool.however for the purposes of the review we have now included both stickler and stylelint.config.js into this repository.

**Review by men32z — APPROVED** [review]
## Good job making the changes, you can merge.



### Status Approved

**Pull request merged**